tests/e2e/specs/layout.cy.js

```diff
@@ -20,7 +20,7 @@ describe('Default layout', () => {
     // visit any page first, so that we create the window.app reference
     cy.visit('/#/workflows/one')
     cy
-      .get('.v-alert')
+      .get('.v-snack__wrapper')
```

Comment:
One thing to try could be giving the snackbar component a unique `data-cy` attribute e.g. "alert-snack" and targeting `[data-cy=alert-snack] > .v-snack__wrapper'